ingestion-core/src/test/java/com/mozilla/telemetry/ingestion/core/util/JsonTest.java

```diff
@@ -24,16 +27,81 @@ public void serialize(JsonGenerator jg, SerializerProvider provider)
   };
 
   @Test
-  public void asStringThrowsUnchecked() {
+  public void asStringCanSucceed() {
+    Json.asString(Json.createObjectNode());
```

Comment:
The tests for Json are at 97.67%, it needs one more test where the input value is an Object.